Dashboard header

mikepadiernos opened the pull request:
### Location:
- HeaderDashboard.js
- Header.js
- UserDashboard

### Details:
Separation of UserDashboard & PinnedCities initiated;
Header for secure view  refactored to present application UX 

### Commited Features:
updated logo & user menu 
initiated separation of userDashboard & PinnedCities

mikepadiernos committed:
refactor(search, user info): components

- added alternative to search component
- refactored presentation to user info

mikepadiernos committed:
refactor(components): detach logo

- moved logo as separate component

mikepadiernos committed:
refactor(components): logo

- added logo alternative image
- converted site title to text from image

mikepadiernos committed:
refactor(components): logo

- added logo alternative image
- converted site title to text from image

mikepadiernos committed:
feature [menu] / item

- added new menu item for "Saved Cities"

mikepadiernos committed:
feature [components] / userDashboard, pinnedCities

- started process in separation of userDashboard & pinnedCities pages
- added route & menu items to point to the pages

mikepadiernos committed:
feature [components] / pinnedCities

- cloned userDashboard for pinnedCities page

mikepadiernos committed:
Merge branch 'main' into DashboardHeader

mikepadiernos committed:
Merge branch 'main' of github.com:Lambda-School-Labs/PT17_cityspire-a-fe into PinnedCities

TheTrabin reviewed (commented):
There's a few test errors with unused variables that should either be commented out or removed, but otherwise well build out.

Unsure if it was intended, but PinnedCities itself isn't called in App.js (App.js line 15 and App.js line 49?).

Markdown was called and commented out in List.stories.js, and it's tossing an unused error (List.stories.js Line 1 and Line 9).

mikepadiernos commented:
I don't recall what the reason for the change in list stories but for the pinned cities I pushed another commit to another branch

mikepadiernos committed:
chore [housekeeping] / prep for merge

- cleaned up some commented code
- added alternative logo
- fixed pinnedCities in App.js

mikepadiernos committed:
Merge remote-tracking branch 'origin/DashboardHeader' into DashboardHeader

bobbyhalljr commented on src/components/common/LogoComponent.js:
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import cityspireLogo from '../../assets/imgs/cityspireLogoAlternative.png';
```
Please remove the unused variables. thanks!

mikepadiernos replied:
Done!

emfresh2o replied:
You really put an effort in creating a new logo, good job!

mikepadiernos committed:
chore [housekeeping] / prep for merge

- cleaned up some commented code

TheTrabin reviewed (approved):
No errors returning. Looks good

emfresh2o reviewed (approved):
I like how you were able to change the logo, good job!
